src/js/index.js

```diff
@@ -62,53 +60,48 @@ const controlOpp = async () => {
     
 };
 
-const controlContacts = async () => {
-    //1) Get contact list(array)
-    
-    await state.contacts.getContacts();
-    
-    //2) Prepare UI(optional)
-
-    //3) Render contacts on UI
-    contactsView.renderContacts(state.contacts.result);
-
-
-    //1) Get chatHistory(array) and profile
-
-    //2) Prepare UI(clear field)
+const controlContacts = async (state) => {
+    // render Profile UI
+    await state.messages.getMessages();

```

Comment:
Pulling data is duplicated for contacts and chat, better to pull once and store locally.